java/src/jmri/server/json/power/JsonPowerHttpService.java

```diff
@@ -37,7 +37,7 @@ public JsonPowerHttpService(ObjectMapper mapper) {
 
     @Override
     // Nullable to override inherited NonNull requirement
-    public JsonNode doGet(String type, @Nullable String name, JsonNode parameters, Locale locale, int id) throws JsonException {
+    public JsonNode doGet(String type, @CheckForNull String name, JsonNode parameters, Locale locale, int id) throws JsonException {
```

Comment:
This must be Nullable per https://static.javadoc.io/com.google.code.findbugs/jsr305/3.0.2/javax/annotation/Nullable.html and http://findbugs.sourceforge.net/manual/annotations.html since this is overriding a Nonnull annotation, and is called out for use in that manner by those documents.

Reply:
The linked docs are also consistent with using @CheckForNull here: "Use CheckForNull to indicate that the element value should always be checked for a null value."